Spigot-Server-Patches/0239-Update-Netty-to-4.1.x.patch

```diff
@@ -0,0 +1,453 @@
+From a6ee7874044ab88681f1c3d717e17660bc214e8a Mon Sep 17 00:00:00 2001
+From: RoccoDev <hey@rocco.dev>
+Date: Sat, 11 Mar 2023 14:33:39 +0100
+Subject: [PATCH] Update Netty to 4.1.x
+
+
+diff --git a/pom.xml b/pom.xml
```

Comment:
I believe we overwrite this file in our CI with `/server_ci_pom.xml`. Could you please also update that file?